src/main/resources/application.properties

```
@@ -0,0 +1,8 @@
+spring.datasource.url= jdbc:postgresql://localhost:5432/hogwarts
+spring.datasource.username= student
+spring.datasource.password= chocolatefrog
+
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto= update
+
+path.to.avatars.folder= /avatars
```

```suggestion
path.to.avatars.folder=./avatars
```
в начале должна быть точка, потому что путь относительный - относительно текущей папке, а не в корне должна быть папка avatars